### PR TITLE
Add test to patch the domain with updated base image in a running AI domain

### PR DESCRIPTION
**Pull request by maggiehe00**
Single ItMiiAuxiliaryImage Jenkins job passed at:
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/5626/testReport/
Jenkins full suite with ItMiiAuxiliaryImage passed at:
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/5627/testReport/

### CODE REVIEW

**Comment by maggiehe00** [comment]
@anpanigr Using this auto test I also verified to use the newer base image tag, 14.1.1.0-11 by simple one line change 
String imageTag = WLS_UPDATE_IMAGE_TAG; 
(right now,  public static final String WLS_UPDATE_IMAGE_TAG = "14.1.1.0-11")

**Review by vanajamukkara — APPROVED** [review]
LGTM

**Review by marinakog — APPROVED** [review]

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiAuxiliaryImage.java [hunk 1]
```diff
@@ -300,11 +301,75 @@ public void testUpdateDataSourceInDomainUsingAuxiliaryImage() {
...
+    logger.info("Currently the image name used for the domain is: {0}", imageName);
+
+    //change image name to imageUpdate
+    String imageTag = TestUtils.getDateAndTimeStamp();
+    String imageUpdate = KIND_REPO != null ? KIND_REPO
+        + (WEBLOGIC_IMAGE_NAME + ":" + imageTag).substring(TestConstants.BASE_IMAGES_REPO.length() + 1)
+        : WEBLOGIC_IMAGE_NAME + ":" + imageTag;
+    dockerTag(imageName, imageUpdate);
+    dockerLoginAndPushImageToRegistry(imageUpdate);
+
```
Comment (anpanigr) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiAuxiliaryImage.java:338]: Here we need to create a new Auxiliary Image with new WebLogic binary ( tagged ) not just patch the spec/image

**Review by anpanigr — APPROVED** [review]
LGTM

**Pull request merged**